__tests__/integration/saveIncompleteRDF.test.js

```diff
@@ -15,7 +15,12 @@ describe('Previewing the RDF', () => {
   })
 
   it('builds the rdf and displays validation errors after attempting to save', async () => {
-    expect.assertions(6) // An additional assertion is done in incompleteFieldsForBibframeInstance
+    expect.assertions(8) // An additional assertion is done in incompleteFieldsForBibframeInstance
+
+    // Need to change something
```

Comment:
by which this means ... this test now is changing something in order to proceed?

Reply:
Reworded this comment to clarify.